code/game/atoms_movable.dm

```diff
@@ -23,8 +23,16 @@
 
 
 /atom/movable/Destroy()
-	..()
+	if(reagents)
+		qdel(reagents)

```

Comment:
Reagents need to be cleaned up by whatever type they're defined on, even if that type should never have reagents. Either move the `reagents` var to atom/movable, or put this back in in atom's Destroy proc.

Also, this should null out reagents. It *shouldn't* need to, since the reagents datum *should* do that itself, but it's best to always explicitly null out references regardless.